python/cugraph-pyg/cugraph_pyg/nn/conv/base.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+from typing import Any, Optional, Tuple, Union
+
+from cugraph.utilities.utils import import_optional
+
+torch = import_optional("torch")
+torch_geometric = import_optional("torch_geometric")
+
+try:  # pragma: no cover
+    from pylibcugraphops.pytorch import (
+        BipartiteCSC,
+        SampledCSC,
+        SampledHeteroCSC,
+        StaticCSC,
+        StaticHeteroCSC,
+    )
+
+    HAS_PYLIBCUGRAPHOPS = True
+except ImportError:
+    HAS_PYLIBCUGRAPHOPS = False
+
+
+class BaseConv(torch.nn.Module):  # pragma: no cover
+    r"""An abstract base class for implementing cugraph-ops message passing layers."""
+
+    def __init__(self):
+        super().__init__()
+
+        if HAS_PYLIBCUGRAPHOPS is False:
+            raise ModuleNotFoundError(
```

Comment:
You should also actually check for >= 23.04

Reply:
We did check this implicitly via
```
try:  # pragma: no cover
    from pylibcugraphops.pytorch import (
        BipartiteCSC,
        SampledCSC,
        SampledHeteroCSC,
        StaticCSC,
        StaticHeteroCSC,
    )
```
as these structures are not available <23.04.

Do you prefer to explicitly check the version?